jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-helpers.c

```diff
@@ -90,8 +90,7 @@ ecma_typedarray_helper_get_shift_size (uint8_t builtin_id) /**< the builtin id o
 #endif /* CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT64 */
     default:
     {
-      JERRY_ASSERT (false);
-      return ECMA_BUILTIN_ID__COUNT;
+      JERRY_UNREACHABLE ();
```

Comment:
This might cause warnings with certain compilers, if the `JERRY_UNREACHABLE` defined as empty. What about the windows port? I think it should complain about the missing return statements.

Reply:
Thanks for the explanation. It is good for me if all of the supported compilers deal with this.